includes/Stories_Renderer/Renderer.php

```diff
@@ -469,28 +491,18 @@ class="<?php echo esc_attr( $single_story_classes ); ?>"
 	 */
 	protected function render_story_with_poster() {
 
-		$story_data            = $this->current();
-		$poster_url            = ( 'circles' === $this->get_view_type() ) ? $story_data->get_poster_square() : $story_data->get_poster_portrait();
-		$poster_style          = sprintf( 'background-image: url(%1$s);', esc_url_raw( $poster_url ) );
-		$inner_wrapper_classes = 'web-stories-list__inner-wrapper ';
-
-		if ( true === $this->is_view_type( 'carousel' ) ) {
-			$poster_style .= sprintf( ' min-width: %1$dpx;', $this->width );
-		}
-
-		if ( ! empty( $this->attributes['list_view_image_alignment'] ) ) {
-			$inner_wrapper_classes .= sprintf( 'image-align-%1$s', $this->attributes['list_view_image_alignment'] );
-		}
+		$story_data   = $this->current();
+		$poster_url   = ( 'circles' === $this->get_view_type() ) ? $story_data->get_poster_square() : $story_data->get_poster_portrait();
+		$poster_style = sprintf( 'background-image: url(%1$s);', esc_url_raw( $poster_url ) );
 
 		?>
-		<div class="<?php echo esc_attr( $inner_wrapper_classes ); ?>">
-			<div
-				class="web-stories-list__story-placeholder"
-				style="<?php echo esc_attr( $poster_style ); ?>"
-			></div>
-			<?php $this->get_content_overlay(); ?>
-		</div>
+
+		<div
+			class="web-stories-list__story-placeholder"
+			style="<?php echo esc_attr( $poster_style ); ?>"
+		></div>
```

Comment:
Doesn't this need a height and width? Why are we using div like this? Why not an img tag?

Reply:
We are not using the actual poster dimensions here, AMP story poster ratio is some 3:4, which looks bit boxy and as per the designs received the ratio used there is 9:16. We are using pseudo elements on `div.web-stories-list__story-placeholder` to create that space and set the image as background.

Also this let's us change the ratio for the `list` view where the image is more boxy.

Another reason is that AMP replaces `img` with `amp-img` with dimension attributes which may not be in line with our design ratio. This causes image to look small.

With this we may need to add the pseudo element ( for the gradient ) on the parent element but then we may not be able to listen to the click event to show the lightbox.

Reply:
This is one way of doing this other would be wrapping the `img` in a div and applying the styles on that for the ratio ( using padding-top/bottom ) and all and positioning `img` absolutely.

Reply:
I haven't looked into this, but is this something the `object-fit` CSS property would make sense to use?

Reply:
that would with a wrapper div on img, 

something like
`
<div class="web-stories-list__story-poster">
    <img src="/image.png" />
</div>
`

then we can make the `img` absolutely positioned and make it fill the parent element with `object-fit`.

Reply:
Would be great if we could make it work like that 👍

Reply:
I've updated the PR to use given markup, since there were issue with the AMP version, AMP adds `layout='intrinsic` for `img` while converting to `amp-img`, which was making image not span full width. So I've checked for for AMP request and used `amp-img` with required attrs.

> https://github.com/rtCamp/web-stories-wp/blob/update/block-styles/includes/Stories_Renderer/Renderer.php#L498